src/components/basic/DlGrid/types.ts

```diff
@@ -2,3 +2,4 @@ export interface DlGridLayout {
     name: string
     value: number[][]
 }
+export type sideType = 'left' | 'right'
```

Comment:
add a DlGrid prefix to sideType so when its exported its somewhat under a namespace